src/java/org/apache/cassandra/io/util/FileUtils.java

```diff
@@ -67,23 +71,84 @@
     public static final boolean isCleanerAvailable;
     private static final AtomicReference<Optional<FSErrorHandler>> fsErrorHandler = new AtomicReference<>(Optional.empty());
 
+    private static Class clsDirectBuffer;
+    private static MethodHandle mhDirectBufferCleaner;
+    private static MethodHandle mhCleanerClean;
+    private static MethodHandle mhDirectBufferAddress;
```

Comment:
`mhDirectBufferAddress` isn't used anywhere. can we eliminate it?

Reply:
done